### PR TITLE
Lvnod-69 create dashboard component and events list 

### PR DESCRIPTION
**Pull request by yuliannakorol**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LVNOD-69)

## Code reviewers

- [ ] @AlbaMontis
- [ ] @shyyka
- [ ] @avoytovich

### Second Level Review

- [ ] @lhalam
- [ ] @4asof4ik

## Summary of change

- added Dashboard component
- added EventsList  container
- created events list reducers

## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR

### CODE REVIEW

File: app/src/components/Dashboard/Dashboard.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Nav, NavItem, Tab, Row, Col } from 'react-bootstrap';
+import { LinkContainer } from 'react-router-bootstrap';
+import EventsList from '../../containers/Events/EventsList';
```
Comment (AlbaMontis) [inline app/src/components/Dashboard/Dashboard.js]: import { EventList } from '../../containers'.

File: app/src/components/Dashboard/dashboard.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.dashboard-menu {
+    position: relative; left: 15px; top: 15;
```
Comment (AlbaMontis) [inline app/src/components/Dashboard/dashboard.scss]: 2 spaces for indentation. Every style rule in new line.

File: app/src/containers/Events/eventsList.scss [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.eventsList {
+    display: inline-block;
```
Comment (AlbaMontis) [inline app/src/containers/Events/eventsList.scss]: Fix indentation.

File: app/src/containers/Events/EventsList.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Link } from 'react-router';
+import { ListGroup,ListGroupItem} from 'react-bootstrap';
```
Comment (AlbaMontis) [inline app/src/containers/Events/EventsList.js]: Missed space after ListGroup, .

File: app/src/components/Dashboard/Dashboard.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Nav, NavItem, Tab, Row, Col } from 'react-bootstrap';
```
Comment (AlbaMontis) [inline app/src/components/Dashboard/Dashboard.js]: I can't find where do you use Tab and Row in this file.

**Review by shyyka — DISMISSED** [review]

File: app/src/components/Dashboard/Dashboard.js [hunk 6]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Nav, NavItem, Col } from 'react-bootstrap';
+import { LinkContainer } from 'react-router-bootstrap';
+import style from './dashboard.scss';
```
Comment (AlbaMontis) [inline app/src/components/Dashboard/Dashboard.js]: You can write just 'import from './dashboard.scss';'

File: app/src/containers/Main/Main.js [hunk 7]
```diff
@@ -1,14 +1,30 @@
 import React from 'react';
-import { Navigation } from './../../components';
+import { Navigation, Dashboard } from './../../components';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as loginActions from '../../redux/Login';
```
Comment (AlbaMontis) [inline app/src/containers/Main/Main.js]: You don't use actions in this file.

File: app/src/containers/Main/Main.js [hunk 8]
```diff
@@ -1,14 +1,30 @@
...
       </div>
     );
   }
 }
+
+const mapStatetoProps = state => ({
+  isAuth: state.login.isAuth
+});
+
+const mapDispatchToProps = dispatch => ({
```
Comment (AlbaMontis) [inline app/src/containers/Main/Main.js]: You don't use actions in this file.

File: app/src/redux/EventsReducers.js [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+      return {
+        ...state,
+        events: action.payload
+      };
+    }
+
+    case RETRIEVE_EVENTS_FAILURE: {
+      return {
+        ...state,
+        error: action.payload
```
Comment (AlbaMontis) [inline app/src/redux/EventsReducers.js]: Can't find where do you use this state.

**Review by shyyka — DISMISSED** [review]

**Review by avoytovich — DISMISSED** [review]

File: app/src/containers/Home/Home.js [hunk 10]
```diff
@@ -1,17 +1,15 @@
...
+import { Grid, Row, Col, PageHeader } from 'react-bootstrap';
+import { Login } from '../../components';
 
-class Home extends React.Component {
+export default class Home extends React.Component {
 
   render() {
     return (
       <Grid>
-        <Message />
```
Comment (4asof4ik) [inline app/src/containers/Home/Home.js]: why you drop Message

**Review by 4asof4ik — CHANGES_REQUESTED** [review]

**Review by shyyka — DISMISSED** [review]

File: app/src/containers/Events/EventsList.js [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as eventsActions from '../../redux/EventsReducers';
+import './eventsList.scss';
+
+class EventsList extends React.Component {
+
+  componentWillMount() {
+    this.props.actions.retrieveEvents();
+    this.setState({ eventsList: this.props.eventsList });
```
Comment (AlbaMontis) [inline app/src/containers/Events/EventsList.js]: Delete this.

File: app/src/containers/Events/EventsList.js [hunk 12]
```diff
@@ -0,0 +1,44 @@
...
+  }
+
+  render() {
+    const eventNode = this.props.eventsList.map((item) => {
```
Comment (AlbaMontis) [inline app/src/containers/Events/EventsList.js]: Useless () around item.

**Review by shyyka — DISMISSED** [review]

**Review by AlbaMontis — DISMISSED** [review]

**Review by shyyka — DISMISSED** [review]

**Review by avoytovich — DISMISSED** [review]

**Review by 4asof4ik — APPROVED** [review]

**Review by denisbodnar — APPROVED** [review]

**Review by shyyka — APPROVED** [review]

**Pull request merged**